src/effects/backends/audiounit/audiouniteffectprocessor.h

```diff
@@ -55,7 +55,7 @@ class AudioUnitEffectProcessor final : public EffectProcessorImpl<AudioUnitEffec
             const GroupFeatureState& groupFeatures) override;
 
   private:
-    AudioUnitManager m_manager;
+    AudioUnitManagerPointer m_manager;
```

Comment:
per https://github.com/mixxxdj/mixxx/wiki/Coding-Guidelines pointers should be prefixed with p, so m_pManager.